sway/tree/view.c

```diff
@@ -1199,6 +1199,7 @@ static void view_save_buffer_iterator(struct wlr_surface *surface,
 		saved_buffer->height = surface->current.height;
 		saved_buffer->x = sx;
 		saved_buffer->y = sy;
+		saved_buffer->transform = wlr_output_transform_invert(surface->current.transform);
```

Comment:
Nit: can we do the `wlr_output_transform_invert` call when rendering? This would make the rendering logic more consistent with the regular path:

https://github.com/swaywm/sway/blob/f7e050c58f61a15bf5987e2e3d280a72666f6381/sway/desktop/render.c#L150

Reply:
Ok, I'll just save the current.transform then.